MAINTAINERS

```diff
@@ -1,69 +1,65 @@
+- Mehran Kholdi (SeMeKh)
+- Harsh Thakur (realHarshThakur)
+- Michael Tsymbalyuk (mtzaurus)
 
-#Reviewers
-"Aditya Jain",@z0marlin,Independent #control-plane-maintainers
-"Akhil Mohan",@akhilerm,MayaData #control-plane-maintainers
-"Aman Gupta",@w3aman,MayaData #control-plane-maintainers
-"Arne Rusek",@arne-rusek,MayaData #mayastor-maintainers
-"Ashutosh Kumar",@sonasingh46,MayaData #control-plane-maintainers
-"Blaise Dias",@blaisedias,MayaData #mayastor-maintainers
-"Chris Wldenyer",@chriswldenyer,MayaData #mayastor-maintainers
-"Evan Powell",@epowell101,MayaData #content-maintainers
-"Giri",@gprasath,MayaData #control-plane-maintainers
-"Harsh Thakur",@realHarshThakur,Independent #control-plane-maintainers
-"Jan Kryl",@jkryl,MayaData #cstor-engine-maintainers, mayastor-maintainers
-"Jonathan Teh",@jonathan-teh,MayaData #mayastor-maintainers
-"Mehran Kholdi",@SeMeKh,Hamravesh #control-plane-maintainers
-"Michael Fornaro",@xUnholy,Independent-Raspbernetes #control-plane-maintainers
-"Michael Tsymbalyuk",@mtzaurus,MayaData #mayastor-maintainers
-"Payes Anand",@payes,MayaData,#control-plane-maintainers, jiva-engine-maintainers, cstor-engine-maintainers, mayastor-maintainers
-"Paul Yoong",@paulyoong,MayaData #mayastor-maintainers
-"Prateek Pandey",@prateekpandey14,MayaData #control-plane-maintainers
-"Praveen Kumar G T",@praveengt,Independent #control-plane-maintainers
-"Ranjith R",@ranjithwingrider,MayaData #content-maintainers
-"Sagar Kumar",@sagarkrsd,Independent #content-maintainers
-"Sai Chaithanya",@mittachaitu,MayaData #control-plane-maintainers
-"Sathya Seelan",@nsathyaseelan,MayaData #control-plane-maintainers
-"Sebastien Dionne",@survivant,Jerabi inc. #control-plane-maintainers
-"Shubham Bajpai",@shubham14bajpai,MayaData #control-plane-maintainers
-"Sjors Gielen",@sgielen,Independent #control-plane-maintainers, cstor-engine-maintainers
-"Tiago Castro",@tiagolobocastro,MayaData #mayastor-maintainers
-"Uma Mukkara",@umamukkara,MayaData #content-maintainers
-"Utkarsh Mani Tripathi",@utkarshmani1997,Independent #control-plane-maintainers, jiva-engine-maintainers
-"Yashpal Choudhary",@iyashu,Independent #control-plane-maintainers
+# Emeritus Maintainers
+#  Name (GitHub Name)
+- Amit Kumar Das (AmitKumarDas) 
+- Karthik Satchitanand (ksatchit)
+- Mayank Patel (mynktl)
+- Murat Karslioglu (muratkars)
+- Pawan Prakash Sharma (pawanpraka1)
+- Richard Elling (richardelling)
+- Vishnu Itta (vishnuitta)
 
-#Emeritus Maintainers or Reviewers due to change in project priorities. 
-#"Peeyush Gupta",@Pensu,DigitalOcean #control-plane-maintainers
-#"Satbir Singh satbirchhikara",@satbirchhikara,MayaData #cstor-engine-maintainers
+# Emeritus Reviewers
+#  Name (GitHub Name)
+- Payes Anand (payes)
+- Shubham Bajpai (shubham14bajpai)
+- Sai Chaithanya (mittachaitu)
+- Sjors Gielen (sgielen)
+- Peeyush Gupta (Pensu)
+- Jan Kryl" (jkryl)
+- Ashutosh Kumar (sonasingh46)
+- Praveen Kumar G T (praveengt)
+- Sagar Kumar (sagarkrsd)
+- Akhil Mohan (akhilerm)
```

Comment:
@akhilerm is still involved in the projects. 

A good way to check who is currently reviewing is to check the stats with last one year. 
https://openebs.devstats.cncf.io/d/75/prs-mergers-table?orgId=1&var-period_name=Last%20year&var-repogroup_name=All